### PR TITLE
[MKC-1011] [UNO R4 WiFi] VRTC & Off Pin tutorial

### PR DESCRIPTION
**Pull request by jacobhylen**
## What This PR Changes
- Adds a short and to the point tutorial on how to use the VRTC & OFF pins on the UNO R4 WiFi

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by karlsoderby — DISMISSED** [review]
@jacobhylen great start, but I think we need to address the following:
- for the VRTC pin, I think we need to show an example sketch. RTC examples have a `startTime()` initiated when the board starts, so this will probably overwrite the current time. If the purpose is to keep the time during power off, we need to show a method of obtaining it after re-launch.
- circuit with batteries needs just a label on which pins we are connecting to.
- a circuit / render that shows precisely how to short GND with OFF (like [this one](https://docs.arduino.cc/tutorials/nano-rp2040-connect/rp2040-01-technical-reference#board-not-detected)). 
- (optional) create a circuit with a power switch connected to OFF pin.

File: content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+title: 'Arduino UNO R4 WiFi VRTC & OFF Pins'
+description: 'Learn how to use the VRTC and OFF Pins on the UNO R4 WiFi.'
+tags:
+  - VRTC
+author: 'Jacob Hylén'
+---
+
+The Arduino UNO R4 WiFi features 2 pins that have not been seen before on UNO boards, the VRTC pin and the OFF pin. 
+
+They are used to control some of the boards functions electrically. The VRTC pin can be used to keep the onboard RTC (Real Time Clock) running even when the boards main power supply is turned off, and the OFF pin is used to turn off the board by cutting off the power.
```
Comment (BenjaminDannegard) [inline content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md:11]:
```suggestion
They are used to control some of the boards electrical functions. The VRTC pin can be used to keep the onboard RTC (Real Time Clock) running even when the boards main power supply is turned off, and the OFF pin is used to turn off the board by cutting off the power.
```

Comment (BenjaminDannegard) [reply content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md:11]: Seemed weirdly worded, but if it is more technically correct then disregard this comment.

File: content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+However, it will only turn off the board when it is powered through the VIN pin, or the barrel jack. Why is this? Because by using this pin, you are turning off the step down converter that generates 5 V from whatever voltage you are providing it with. If you are powering the board from USB, 5 V is provided from the USB cable, and there is no need for this step down converter to begin with. 
+
+To use the OFF pin, all you need to do is to create a short circuit from it to a GND connection, like in the diagram below. To experiment, you can do this with a jumper cable, but for your finished projects you may want to incorporate a button or switch that will turn the board on or off in this way.
+
+![OFF Pin Shorted to GND](./assets/OFF.png)
+
+## Summary
+
+This short tutorial shows how to use the VRTC and OFF pins that are found on the new header on the Arduino UNO R4 WiFi, features that are brand new to the UNO family.
```
Comment (BenjaminDannegard) [inline content/hardware/02.hero/boards/uno-r4-wifi/tutorials/vrtc-off/vrtc-off.md:85]:
```suggestion
This short tutorial showed how to use the VRTC and OFF pins that are found on the new header on the Arduino UNO R4 WiFi. These features are brand new to the UNO family.
```

**Review by BenjaminDannegard — COMMENTED** [review]
Looks good! Just two small notes

**Review by BenjaminDannegard — APPROVED** [review]
LGTM

**Pull request merged**